aiida/backends/sqlalchemy/models/computer.py

```diff
@@ -31,13 +32,25 @@ class DbComputer(Base):
     __tablename__ = 'db_dbcomputer'
 
     id = Column(Integer, primary_key=True)  # pylint: disable=invalid-name
-    uuid = Column(UUID(as_uuid=True), default=get_new_uuid, unique=True)
-    label = Column(String(255), unique=True, nullable=False)
-    hostname = Column(String(255))
-    description = Column(Text, nullable=True)
-    scheduler_type = Column(String(255))
-    transport_type = Column(String(255))
-    _metadata = Column('metadata', JSONB)
+    uuid = Column(UUID(as_uuid=True), default=get_new_uuid, nullable=False)
+    label = Column(String(255), nullable=False)
```

Comment:
does this not need a default?

Reply:
There is no default on the django side. I imagine we should not be "silently" setting these labels to a default, because not specifying a label is likely to be a bug of the upstream code